docs/source/modules/tutorial.rst

```diff
@@ -0,0 +1,653 @@
+
+What does Brain-Score look like, exactly?
+--------------------------------------------------------------------
+The main code for Brain-Score is contained and hosted on Github (https://github.com/brain-score)
+on various repos, free for anyone to fork,
+clone, or download. The main page has 10 repos (9 visible to non-devs)
+that make up the entire source code base- this can be overwhelming, but
+fear not. I will walk you through what each does, as well as explain that
+you most likely will only need one to get a model up and scored. It is
+good to see, however, the structure of how Brain-Score operates with its
+various repos. They are:
+
+1. ``brain-score``: the heart of the code that runs analysis and comparisons.
+2. ``sample-model-submission``: template and examples for model submissions.
+3. ``candidate_models``: various pre-trained models/models that you can submit.
+4. ``model-tools``: helper functions that translate from machine learning models
+   to brain models to be tested on brain-score.
+5. ``brainio_collection``: the repo that packages and collects the stimuli/data.
+6. ``brainio_base``: repo that contains various data structures for ``BrainIO``.
+7. ``result_caching``: a helper repo to store the results of function calls so they can
+   be re-used without re-computing.
+8. ``brain-score.web``: website front and back end.
+9. ``Brainio_contrib`` (archived): used in the past to contribute stimuli and datasets
+   (now part of ``brainio_collection``).
+
+Which repos will I use?
+-----------------------
+When we get to the install guide, I will show you exactly how to
+clone/fork repos for your own project in the easiest way possible.
+But for now, you will mainly only need the ``sample-model-submission`` repo.
+
+How do I get a Brain-Score for my model?
+----------------------------------------
+
+Brain-Score currently has two “ways" to score your model:
+
+1. *On your machine*:  your model is scored on one or more benchmarks (a benchmark is a
+   standard to compare against, like Majaj-Hong’s 2015 paper on IT results (which you
+   can read more about here: https://brain-score.readthedocs.io/en/latest/modules/benchmarks.html). This is useful if you would like to test and see if your
+   model is ready to be submitted and run against all benchmarks, or if you want a quick
+   and dirty way to score on a single benchmark, like V4 or IT. This is mainly used to
+   test and/or optimize your model before submitting to the main site.
+2. *Remote*: your model is run on Brain-Score’s website, and contains scores against all the
+   local benchmarks, plus many others (we avoid sharing this code with users to
+   avoid overfitting). The (hierarchical) mean of all these benchmark scores is the
+   Brain-Score itself. This has more overhead to get started, but is worth it, as you
+   can set up pull requests to our main code package (i.e, suggest changes) if you find
+   errors and have the packages/dependencies automatically installed.
```

Comment:
->
"Submit your model on the Brain-Score website to test it on all available benchmarks. Note that a submission is the only way to score models on private evaluation data. The average score will be the hierarchical mean of the individual benchmark scores." (drop the rest I think, PRs to the main code-base don't belong in the submission description)